packages/ui/globals.scss

```diff
@@ -140,3 +140,11 @@ body {
   --red: #d4351c;
   --text-grey: #0b0c0c;
 }
+// allows cookie banner to stick to bottom of veiw port
+.SE-cookie-banner{
+  bottom: 0;
+  position: fixed;
```

Comment:
_Suggestion:_ Consider 'position: sticky;' for this type of functionality.

https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky

Reply:
having looked at the docs Im not sure sticky would work well for SE,
within the document flow the cookie banner is one of the first elements within the app, this is done for users that navigate by keyboard to quickly and easily access the banner.

I based its position on the banners I had done previously (BPoR and VS) which are at the bottom however FRF is at the top, this should be part of the wider conversation on cookie banners across our apps.